gloox 1.0.14 broke ABI compatibility, so we need to bump soversion. Upstream
has confirmed that this should be done and was simply forgotten last release.

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -40,7 +40,7 @@
                         carbons.cpp jinglepluginfactory.cpp jingleiceudp.cpp jinglefiletransfer.cpp \
                         iodata.cpp
 
-libgloox_la_LDFLAGS = -version-info 13:0:0 -no-undefined -no-allow-shlib-undefined
+libgloox_la_LDFLAGS = -version-info 14:0:0 -no-undefined -no-allow-shlib-undefined
 libgloox_la_LIBADD =
 libgloox_la_CFLAGS = $(CPPFLAGS)
 
